src/SeqCli/Cli/Features/BatchSizeFeature.cs

```diff
@@ -0,0 +1,19 @@
+﻿using SeqCli.Util;
+
+namespace SeqCli.Cli.Features
+{
+    class BatchSizeFeature: CommandFeature
+    {
+        const int DefaultBatchSize = 100;
+        int? _value;
+
+        public int Value => _value ?? DefaultBatchSize;
+
+        public override void Enable(OptionSet options)
+        {
+            options.Add("batchsize=",
```

Comment:
`seqcli`'s arguments use kebab-case for compound names - `batch-size` would be a better fit here

Reply:
Ahh it looks like we've got a bit of a mix, with some options like `apikey` and `invalid-data`.

I think `batch-size` is nicer though :+1: